lms/envs/common.py

```diff
@@ -1529,8 +1529,16 @@
 
 
 common_js = set(rooted_glob(COMMON_ROOT / 'static', 'coffee/src/**/*.js')) - set(courseware_js + discussion_js + notes_js + instructor_dash_js)    # pylint: disable=line-too-long
-project_js = set(rooted_glob(PROJECT_ROOT / 'static', 'coffee/src/**/*.js')) - set(courseware_js + discussion_js + notes_js + instructor_dash_js)  # pylint: disable=line-too-long
-
+project_js = (
+    set(
+        rooted_glob(PROJECT_ROOT / 'static', 'coffee/src/**/*.js') +
+        [
+            'common/js/xblock/runtime.v1.js',
+            'lms/js/xblock/lms.runtime.v1.js',
+        ]
+    )
+    - set(courseware_js + discussion_js + notes_js + instructor_dash_js)
```

Comment:
hm, not a huge fan of splitting a math expression between lines - can we assign each half to a variable and do the subtraction in a third step?